test/uk/gov/hmrc/eventhub/repository/SubscriberQueueRepositorySpec.scala

```diff
@@ -27,16 +28,31 @@ import uk.gov.hmrc.eventhub.model.TestModels.{channelPreferences, event}
 import uk.gov.hmrc.mongo.MongoComponent
 import play.api.libs.ws.WSClient
 import uk.gov.hmrc.integration.ServiceSpec
-import uk.gov.hmrc.mongo.workitem.ProcessingStatus.{Failed, InProgress, PermanentlyFailed, ToDo}
+import uk.gov.hmrc.mongo.workitem.ProcessingStatus.{InProgress, PermanentlyFailed, ToDo}
 
 import java.time.Instant
 import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
 
 class SubscriberQueueRepositorySpec
     extends AnyFlatSpec with Matchers with IdiomaticMockito with ScalaFutures with ServiceSpec {
 
+  lazy val ttlInSecondsEvent = 300
+  lazy val ttlInSecondsSubscribers = 300
+
   override def externalServices: Seq[String] = Seq.empty[String]
 
+  override def additionalConfig: Map[String, _ <: Any] =
```

Comment:
could be just make type to be _ or just Any to make it simple ?

Reply:
the type definitation comes from here: https://github.com/hmrc/service-integration-test/blob/main/src/main/scala/uk/gov/hmrc/integration/ServiceSpec.scala#L54

Reply:
the upper bound type def is clear I think, I don't disagree with the explicitness